.github/workflows/mega-linter-for-runner.yml

```diff
@@ -43,11 +43,11 @@ jobs:
       - name: MegaLinter
         id: ml
         # You can override MegaLinter flavor used to have faster performances
-        # More info at https://megalinter.io/flavors/
+        # More info at https://megalinter.io/latest/flavors/
         uses: oxsecurity/megalinter/flavors/javascript@beta
         env:
           # All available variables are described in documentation
-          # https://megalinter.io/configuration/#shared-variables
+          # https://megalinter.io/latest/configuration/
```

Comment:
Was there a problem with the anchor link, that referred to a specific section?

Reply:
Yes, this anchor doesn't exist. Instead of removing the anchor I just replaced it with another: #common-variables.